config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-presubmits.yaml

```diff
@@ -52,7 +47,9 @@ presubmits:
           set -o pipefail;
           set -o xtrace;
           REPO_ROOT=$(git rev-parse --show-toplevel);
-          cd "${REPO_ROOT}/../kubetest2" || exit 1;
-          make install;
-          make install-deployer-gce;
+          cd;
+          export GO111MODULE=on;
+          go get sigs.k8s.io/kubetest2@latest;
```

Comment:
you can do `go get sigs/k8s.io/kubetest2/...` if they're the same module I think

Reply:
FWIW doing so would install all deployers and testers, yes?

Reply:
(not just the ones we need)

Reply:
good point. a concern though is trying to make this easy to replicate locally.

in the future we should not be using HEAD (use a tagged vesion) and we should expect users to be able to user this locally.

Reply:
I'd expect most upstream deployers to be cheap to install or moved out of the main module.

Reply:
As far as using HEAD goes, we avoid this problem with the standard e2e workflow by using tagged versions of kubekins with a certain build of kubetest. Autobumper solves updating versions. We don't yet have an equivalent for kubetest2. @amwat mentioned wanting to put together a much smaller e2e test container that provides kubetest2 with similar semantics.

For install, I agree it should be easy. Happy to go either way with it. The eventual new test container should solve this problem completely.

Reply:
well this job is meant to be at head intentionally so we want this either way. but yes eventually we can go get tags and or reuse what's baked in the test image. but probably go getting tags is more reproducible anyway.

Reply:
@michaelmdresser let's also get a dupe of this job running against kubetest2 as a presubmit.